lumeer-core/src/main/java/io/lumeer/core/facade/UserNotificationFacade.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.lumeer.core.facade;
+
+import io.lumeer.api.model.Permission;
+import io.lumeer.api.model.ResourceType;
+import io.lumeer.api.model.UserNotification;
+import io.lumeer.api.model.common.Resource;
+import io.lumeer.core.WorkspaceKeeper;
+import io.lumeer.core.auth.AuthenticatedUser;
+import io.lumeer.engine.api.data.DataDocument;
+import io.lumeer.engine.api.event.UpdateResource;
+import io.lumeer.storage.api.dao.UserNotificationDao;
+
+import java.time.ZonedDateTime;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.enterprise.context.RequestScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+/**
+ * @author <a href="mailto:marvenec@gmail.com">Martin Večeřa</a>
+ */
+@RequestScoped
+public class UserNotificationFacade extends AbstractFacade {
+
+   private Logger log = Logger.getLogger(UserNotificationFacade.class.getName());
```

Comment:
Wouldn't it be better to declare this logger as static field? According to [SLF4J FAQ](https://www.slf4j.org/faq.html#declared_static) there is less CPU and memory overhead. Because a new instance of this facade is created with every request.